vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -10469,6 +10473,14 @@ quantitykind:MolalityOfSolute
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
   rdfs:label "Molality of Solute"@en ;
 .
+quantitykind:AmountOfSubstancePerUnitMassPressure
```

Comment:
Wow, this blows my mind. The mass of a mole changes under pressure? I think I believe you, but what is going on here?